app/Models/Category.php

```diff
@@ -14,7 +14,7 @@ class Category extends Model
         'title',
     ];
 
-    public function products(): HasMany
+    public function property(): HasMany
```

Comment:
Also you changed this, I get why, but it'll affect where i called this function in the resource since you didn't change it there